docs/ref-supported.md

```diff
@@ -59,6 +59,7 @@ IAXU | ITS: XML Templates for HTML Templates | Yes
 IDOC | IDoc Type | Yes
 IEXT | Enhancement | Yes
 INTF | Interface (ABAP objects) | Yes
+IOBJ | BW: InfoObject | [#2921](https://github.com/larshp/abapGit/issues/2921)
```

Comment:
please set this to "Yes" instead of the link, so it reflects that this is supported after merge of this PR

Reply:
Done